src/fidesops/schemas/base_class.py

```diff
@@ -20,3 +20,13 @@ class Config:
 
 
 BaseSchema = FidesopsSchema
+
+
+class DocsOnlySchema(BaseModel):
```

Comment:
The function of the class isn't anything to do with the docs, would `NoValidationSchema` be a better name here? It's great that we say explicitly in the docstring that validation is switched off too.

Reply:
Yes that makes sense to me